c10/core/SymInt.h

```diff
@@ -94,7 +94,7 @@ class C10_API SymInt {
   SymIntNodeImpl* toSymIntNodeImplUnowned() const {
     TORCH_INTERNAL_ASSERT_DEBUG_ONLY(is_symbolic());
     uint64_t unextended_bits = static_cast<uint64_t>(data_) & ~MASK;
-    uint64_t sign_bit_mask = 1ULL << (61 - 1);
+    uint64_t sign_bit_mask = 1ULL << (62 - 1);
```

Comment:
QQ: what's the context for the change to `62`? 

IIRC, we chose `61` to support `-1` for ops like `expand` where `-1` has a special meaning.

Reply:
There's no change, there was a previous change but it was a typo.